pr-data.csv

```diff
@@ -947,8 +947,8 @@ https://github.com/FasterXML/jackson-dataformats-binary,d78d13c574a4b139438841bd
 https://github.com/FasterXML/jackson-dataformats-binary,d78d13c574a4b139438841bd5859b8b99beb63ea,avro,com.fasterxml.jackson.dataformat.avro.interop.annotations.AvroEncodeTest.testUuidValue[Apache to Jackson with Jackson schema],ID,,,
 https://github.com/FasterXML/jackson-dataformats-binary,d78d13c574a4b139438841bd5859b8b99beb63ea,avro,com.fasterxml.jackson.dataformat.avro.interop.annotations.AvroEncodeTest.testUuidValue[Jackson to Jackson with Apache schema],ID,,,
 https://github.com/FasterXML/jackson-dataformats-binary,d78d13c574a4b139438841bd5859b8b99beb63ea,avro,com.fasterxml.jackson.dataformat.avro.interop.annotations.AvroEncodeTest.testUuidValue[Jackson to Jackson with Jackson schema],ID,,,
-https://github.com/FasterXML/jackson-datatypes-collections,8b88868ac3012783fb7f1bda00309c823172e02a,guava,com.fasterxml.jackson.datatype.guava.FluentIterableTest.testSerialization,ID,,,
-https://github.com/FasterXML/jackson-datatypes-collections,8b88868ac3012783fb7f1bda00309c823172e02a,guava,com.fasterxml.jackson.datatype.guava.FluentIterableTest.testWrappedSerialization,ID,,,
+https://github.com/FasterXML/jackson-datatypes-collections,8b88868ac3012783fb7f1bda00309c823172e02a,guava,com.fasterxml.jackson.datatype.guava.FluentIterableTest.testSerialization,ID,InspiredAFix,https://github.com/FasterXML/jackson-datatypes-collections/commit/9c14eaa4fa6f3c57623e6cb6573fd604928165a0,https://github.com/FasterXML/jackson-datatypes-collections/pull/91
+https://github.com/FasterXML/jackson-datatypes-collections,8b88868ac3012783fb7f1bda00309c823172e02a,guava,com.fasterxml.jackson.datatype.guava.FluentIterableTest.testWrappedSerialization,ID,InspiredAFix,https://github.com/FasterXML/jackson-datatypes-collections/commit/9c14eaa4fa6f3c57623e6cb6573fd604928165a0,https://github.com/FasterXML/jackson-datatypes-collections/pull/91
```

Comment:
@LALAYANG thanks for the changes. For these two lines, please have the PR link be the PR that inspired a fix and the commit be in the Notes column

Reply:
Hi Wing, I made the changes, but the checker still failed

Reply:
To add notes, can you please follow the instructions in the README?
https://github.com/TestingResearchIllinois/idoft#adding-notes

Namely, create an issue report describing the fact that this commit link actually introduces a fix for these two flaky tests.